File: financialplanning/template/financialplanning.py
def solve(options:list, min_to_retire: int) -> int:

    # YOUR CODE HERE

    return 0

if __name__ == '__main__':

    n, M = map(int,input().split(' '))

    options = []
    for _ in range(n):
        options.append( tuple(map(int,input().split(' '))) )
    print( solve(options, M) )

File: financialplanning/template/financialplanning.cpp
#include <iostream>
#include <string>
#include <vector>
using namespace std;

int solve(vector<int>* profits, vector<int>* costs, int min_to_retire)
{
	//YOUR CODE HERE
	return -1;
}

int main(){
	int n, M;
	cin >> n >> M;
	vector<int>* profits = new vector<int>();
	vector<int>* costs   = new vector<int>();
	int p_i, c_i;
	for (int i=0; i<n; i++)
       	{
		cin >> p_i >> c_i;
		profits->push_back( p_i );
		costs->push_back( c_i );
	}

	cout << solve(profits, costs, M) << endl;
	return 0;
}
